src/test/java/se/sundsvall/supportmanagement/integration/db/util/ErrandNumberGeneratorTest.java

```diff
@@ -17,78 +16,90 @@
 import org.mockito.Mock;
 import org.mockito.junit.jupiter.MockitoExtension;
 
+import org.zalando.problem.Problem;
 import se.sundsvall.supportmanagement.integration.db.ErrandNumberSequenceRepository;
+import se.sundsvall.supportmanagement.integration.db.NamespaceConfigRepository;
 import se.sundsvall.supportmanagement.integration.db.model.ErrandNumberSequenceEntity;
+import se.sundsvall.supportmanagement.integration.db.model.NamespaceConfigEntity;
 
 
 @ExtendWith(MockitoExtension.class)
 class ErrandNumberGeneratorTest {
 
 	private static final DateTimeFormatter dateFormatter = DateTimeFormatter.ofPattern("yyMM");
-
 	private static final String MUNICIPALITY_ID = "2281";
-
 	private static final String NAMESPACE = "CONTACTCENTER";
+	private static final String SHORT_CODE = "KC";
+
+
+	@Mock
+	ErrandNumberSequenceRepository repositoryMock;
```

Comment:
private? (comment also applies to `namespaceConfigRepositoryMock` and `stringGeneratorService)`